test/Responses/Models/Address.cs

```diff
@@ -0,0 +1,11 @@
+
+namespace Ovh.Test.Models
+{
+    public class Address
+    {
+        public string city { get; set; }
+        public string country { get; set; }
+        public string line1 { get; set; }
+        public string zip { get; set; }
+    }
```

Comment:
You're defining real (?) return values here : is there any reason not to provide those routes directly inside the client?

Reply:
- While they work, they are only the required properties for requests, and some selected properties for responses, so they wouldn't fit
- The lib is not intended to be a full fledged SDK. It will never embark Models as this requires careful monitoring of API changes